local/kcp/start.sh

```diff
@@ -42,8 +42,8 @@ kcp-binaries() {
     git clone https://github.com/kcp-dev/kcp.git
     KCP_DIR="${KCP_PARENT_DIR}/kcp"
     pushd kcp
-    KCP_BRANCH="${KCP_BRANCH:-release-0.7}"
-    git checkout "${KCP_BRANCH}"
+    KCP_TAG="${KCP_BRANCH:-v0.8.0}"
```

Comment:
It is nice not to break the user interface as you did but it is getting a bit confusing. What do you think of renaming the script parameter? The script is only for local dev, isn't it? Is this script called somewhere, in CI maybe?
I am fine either way.

Reply:
Sure, renamed the param to KCP_TAG as well.